### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by Viicos**
Things I noticed over the past few weeks

### CODE REVIEW

File: src/openforms/authentication/service.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
-from .constants import FORM_AUTH_SESSION_KEY, AuthAttribute
+from .constants import (
+    FORM_AUTH_SESSION_KEY,
+    REGISTRATOR_SUBJECT_SESSION_KEY,
```
[the remark this answers is not in the document]
Comment (Viicos) [reply src/openforms/authentication/service.py:3]: We already discussed it here: https://github.com/open-formulieren/open-forms/pull/4305#discussion_r1603525916

[the remark this answers is not in the document]
Comment (Viicos) [reply src/openforms/authentication/service.py:3]: Ah are you talking about the `REGISTRATOR_SUBJECT_SESSION_KEY`?

File: src/openforms/appointments/contrib/jcc/tests/test_plugin.py [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import re
-from datetime import date, datetime
+from datetime import date, datetime, timezone
 
 from django.test import SimpleTestCase, TestCase
-from django.utils import timezone
```
[the remark this answers is not in the document]
Comment (Viicos) [reply src/openforms/appointments/contrib/jcc/tests/test_plugin.py:5]: Hum, it creates more diff when only `timezone.utc` is used. Maybe we should use `from datetime import datetime, timezone as dt_timezone` only when `django.utils.timezone` is used as well?

[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/appointments/contrib/jcc/tests/test_plugin.py:5]: or leave it as it is, we have plenty of time until the deprecation is removed. it doesn't cause problems, does it?

[the remark this answers is not in the document]
Comment (Viicos) [reply src/openforms/appointments/contrib/jcc/tests/test_plugin.py]: Removed, but will have to be tackled during the 5.2 upgrade

[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/appointments/contrib/jcc/tests/test_plugin.py]: Django 4.2 is EOL in two years, we have time.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**